contracts/BdStable/Pools/BdPoolLibrary.sol

```diff
@@ -1,8 +1,7 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.6.12;
-pragma experimental ABIEncoderV2;
+pragma solidity 0.8.13;
```

Comment:
I still see many places we are using SaFeMath... is it by purpose?

Reply:
Duplicated, let's continue the discussion in the original comment.